app/helpers/textual_mixins/textual_devices.rb

```diff
@@ -42,11 +42,11 @@ def disks_attributes
       location = disk.location
       size = disk.size
       prov = disk.used_percent_of_provisioned
-      device_name = _("Hard Disk (%{controller_type} %{location}), Size %{size}, Percent Used Provisioned Space %{space}") %
-                       {:controller_type => ctrl_type,
-                        :location        => location,
-                        :size            => size,
-                        :space           => prov}
+      device_name = _("Hard Disk (%{controller_type} %{location}), Size %{size},
+                      Percent Used Provisioned Space %{space}") % {:controller_type => ctrl_type,
```

Comment:
yea sorry, it could lead to the trouble with fast_gettext, going to use backlash as you mentioned that.